getting-started/mbed_app.json

```diff
@@ -2,7 +2,8 @@
     "target_overrides": {
         "*": {
              "platform.stdio-convert-newlines": true,
-             "target.features_add" : ["EXPERIMENTAL_API"],
+             "target.features_add" : ["EXPERIMENTAL_API", "PSA"],
+             "target.extra_labels_add": ["MBED_PSA_SRV"],
```

Comment:
Do we really need MBED_PSA_SRV? Shouldn't this example work also with a TF-M target?

Reply:
Done